### PR TITLE
Added Avatar(Icons) and Badge

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Please Note the Icons here Represent Avatar

### CODE REVIEW

File: Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    background-color: green;
+    border-radius: 50%;
+    height: 1rem;
+    width: 1rem;
+    position: absolute;
+    bottom: 3%;
+    right: 10%;
+    border: 2px solid white
+}
+.center .badges .badge.badged-avatar.available{
```
Comment (md-amir-gauhar) [inline Badge/badge.css:82]: Try using vars for colors. Else all is fine👍

**Review by md-amir-gauhar — APPROVED** [review]

File: Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,293 @@
+/* *{
```
Comment (Hamza-777) [inline Badge/badge.css:1]: Remove the commented code if not needed. Makes the code cleaner.

File: ComponentsCss/header.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  
+  /* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: #3e8e41;border-radius: 3rem 0rem 0rem 3rem; color: white;margin-top: 0.1rem;text-shadow: 2px 2px 1px black;}
+  
+  /* Show the dropdown menu on hover */
+  .dropdown:hover .dropdown-content {display: block;}
+  
+  /* Change the background color of the dropdown button when the dropdown content is shown */
+  .dropdown:hover .dropbtn {background-color: #3e8e41; border-radius: 3rem 0rem 0rem 3rem; color: white;}
+
```
Comment (Hamza-777) [inline ComponentsCss/header.css:43]: Remove the extra space.

**Review by Hamza-777 — APPROVED** [review]
Make the pointed out changes and you are good to go.

File: Badge/badge.css (hunk 1 above)
Comment (HrishikeshMahalle) [reply Badge/badge.css:82]: Thank You!!!!! Definitely ill have a look into it.

File: ComponentsCss/header.css (hunk 3 above)
Comment (HrishikeshMahalle) [reply ComponentsCss/header.css:43]: Thank you Hamza!!! i'll have a look into it.

**Pull request merged**